src/manage_nsfs/manage_nsfs_cli_utils.js

```diff
@@ -70,8 +70,12 @@ function get_config_file_path(config_type_path, file_name) {
     return path.join(config_type_path, file_name + '.json');
 }
 
-function get_symlink_config_file_path(config_type_path, file_name) {
-    return path.join(config_type_path, file_name + '.symlink');
+function get_symlink_config_file_path(config_type_path, file_name, root_account) {
```

Comment:
The change is needed because by-name symlink is now under root account, so you need both root account name and iam account name. Sometimes they're the same, if you're accessing the root account, but not necessarily.